drape/utils/glyph_usage_tracker.cpp

```diff
@@ -16,14 +16,14 @@ std::string GlyphUsageTracker::GlyphUsageStatistic::ToString() const
   ss << " Invalid glyphs count = " << m_invalidGlyphs.size() << "\n";
   ss << " Invalid glyphs: { ";
   for (auto const & it : m_invalidGlyphs)
-    ss << it.first << "(" << it.second << ") ";
+    ss << std::hex << static_cast<uint32_t>(it.first) << std::dec << "(" << it.second << ") ";
```

Comment:
Is this cast needed if we leave DebugPrint?

Reply:
Same here.